reflex/constants.py

```diff
@@ -56,46 +74,39 @@
 # Min Bun Version
 MIN_BUN_VERSION = "0.7.0"
 # The directory to store the bun.
-BUN_ROOT_PATH = os.path.join(REFLEX_DIR, ".bun")
+BUN_ROOT_PATH = os.path.join(REFLEX_DIR, "bun")
 # Default bun path.
 DEFAULT_BUN_PATH = os.path.join(BUN_ROOT_PATH, "bin", "bun")
 # URL to bun install script.
 BUN_INSTALL_URL = "https://bun.sh/install"
 
-# NVM / Node config.
-# The NVM version.
-NVM_VERSION = "0.39.1"
+# FNM / Node config.
 # The FNM version.
 FNM_VERSION = "1.35.1"
 # The Node version.
 NODE_VERSION = "18.17.0"
 # The minimum required node version.
 NODE_VERSION_MIN = "16.8.0"
-# The directory to store nvm.
-NVM_DIR = os.path.join(REFLEX_DIR, ".nvm")
 # The directory to store fnm.
 FNM_DIR = os.path.join(REFLEX_DIR, "fnm")
+FNM_FILENAME = get_fnm_name()
```

Comment:
Can be moved to `prerequisites.py`

Reply:
The `FNM_FILENAME` is needed to construct the URL to the release binary (`FNM_INSTALL_URL`). Do you mean to move that as well (i.e move `get_fnm_name`, `FNM_FILE_NAME` and `FNM_INSTALL_URL` to prerequisites)?